app/init_test.go

```diff
@@ -8,42 +8,75 @@ import (
 
 func Test_getDefaultNodeHome(t *testing.T) {
 	type testCase struct {
-		name         string
-		userHome     string
-		celestiaHome string
-		want         string
+		name            string
+		userHome        string
+		celestiaHome    string
+		celestiaHomeOld string
+		want            string
 	}
 
 	testCases := []testCase{
 		{
-			name:         "want .celestia-app when userHome is empty and celestiaHome is empty",
-			userHome:     "",
-			celestiaHome: "",
-			want:         ".celestia-app",
+			name:            "want .celestia-app when userHome is empty and celestiaHome/Old are empty",
```

Comment:
```suggestion
			name:            "want .celestia-app when userHome is empty and celestiaHomeOld are empty",
```

Reply:
Oh I meant this was celestiaHome and celestiHomeOld

Reply:
Oh got it, thanks